doc/changes/changes_3.0.0.md

```diff
@@ -0,0 +1,12 @@
+# 3.0.0 - 2025-06-13
```

Comment:
No codename?

Reply:
Is the breaking change that justifies a new major number the move to GitHub?

Reply:
The breaking change is that we removed support for AWS Code Build.